src/elements/content-sharing/SharingNotification.js

```diff
@@ -198,14 +199,15 @@ function SharingNotification({
     ]);
 
     // Set the collaborators list
-    const collaboratorsList: collaboratorsListType | null = useCollaborators(
-        api,
-        itemID,
-        itemType,
-        data => convertCollabsResponse(data, ownerEmail, currentUserID === ownerID),
-        () => handleError(contentSharingMessages.collaboratorsLoadingError),
+    const collaboratorsListFromAPI: Collaborations | null = useCollaborators(api, itemID, itemType, undefined, () =>
+        handleError(contentSharingMessages.collaboratorsLoadingError),
     );
-    if (collaboratorsList && !collabsExist) {
+    if (collaboratorsListFromAPI && !collabsExist) {
```

Comment:
Is the `collabsExist` state needed? Is the truthiness of `collaboratorsListFromAPI` insufficient

Reply:
Yeah, this prevents an infinite loop in which `setCollaboratorsList()` keeps getting called. I updated the component to check for the existence of state values from the parent component.